### PR TITLE
added required infos for Maven Central

### PR DESCRIPTION
**Pull request by kaikreuzer**
Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,6 +22,40 @@
...
+      <name>Kai Kreuzer</name>
+      <email>kai@openhab.org</email>
+      <organization>openHAB</organization>
+      <organizationUrl>http://www.openHAB.org</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:openhab/infrastructure.git</connection>
+    <developerConnection>scm:git:git@github.com:openhab/infrastructure.git</developerConnection>
```
Comment (pfink) [inline pom.xml:42]: Don't wee need `https://` as a url prefix here?

Comment (kaikreuzer) [reply pom.xml]: where do you see a https?

Comment (kaikreuzer) [reply pom.xml:42]: fixed

**Comment by pfink** [comment]
resolves #7

**Pull request merged**